etc/scripts/deps_archive.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import argparse
+from commoncode.system import on_windows
+import os
+from shutil import make_archive
+import sys
+
+
+def generate_OS_archive(input_dir, output):
+    """
+    Generate an archive for dependencies for specific OS and
+    given version of python by taking directory as an input.
+    """
+
+    root_dir = os.path.expanduser(os.path.join("~", input_dir))
+    output_dir = os.path.expanduser(os.path.join("~", output))
```

Comment:
why do you force these to be relative to my home directory? use abspath and expanduser, but do NOT join with `os.path.join("~"`

Reply:
because it is recommended in python documentation of make_archive api.

Reply:
I am not convinced we need these `make_archive` and in anycase we do not want any `os.path.join("~"`... there if we were to create archives